examples/src/bin/limitedclient.rs

```diff
@@ -11,7 +11,7 @@ use rustls::OwnedTrustAnchor;
 
 fn main() {
     let mut root_store = rustls::RootCertStore::empty();
-    root_store.add_server_trust_anchors(
+    root_store.add_trust_anchors(
```

Comment:
Maybe we should do this as an alias on the 0.21.x branch with a `#[deprecated]` to help folks transition?

Reply:
PTAL: https://github.com/rustls/rustls/pull/1383

Reply:
Done: https://github.com/rustls/rustls/releases/tag/v%2F0.21.6